frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/FetchAllRemotesAction.java

```diff
@@ -19,6 +20,16 @@
 @CustomLog
 public class FetchAllRemotesAction extends BaseProjectDependentAction {
 
+  private static final long FETCH_ALL_UP_TO_DATE_TIMEOUT = 60 * 1000;
```

Comment:
Pls also include `_MILLIS` for consistency/clarity (there's https://checkerframework.org/manual/#units-checker but that's maybe an overkill for the time being 😅)